examples/basic/meta.json

```diff
@@ -1,7 +0,0 @@
-{
-  "name": "Next.js",
-  "description": "Minimal Turborepo example for learning the fundamentals.",
-  "template": "https://vercel.com/templates/next.js/turborepo-next-basic",
-  "featured": true,
-  "boost": true
-}
```

Comment:
I think we still need this?

Reply:
Good catch! @anthonyshew we do still need this. This file is read is how we build the list of examples in the docs at build time.

Reply:
More details on how this works:

https://github.com/vercel/turbo/blob/main/docs/pages/repo/docs/getting-started/from-example.mdx#L10-L41

Reply:
Not sure when I did that. :shocked:

Thanks for the catch! @mehulkar to the rescue as usual.